static/js/data_handler.js

```diff
@@ -74,6 +75,46 @@ export let dataHandler = {
     },
     createNewCard: function (cardTitle, boardId, statusId, callback) {
         // creates new card, saves it and calls the callback function with its data
-    }
+    },
     // here comes more features
+    submitNewTitle: function submit_new_title(event) {
+                        let header = event.currentTarget.parentElement;
+                        let newTitle = header.querySelector(".new-board-title");
+                        console.log(newTitle.value);
```

Comment:
🔪 (I always comment a knife (or swords) when I think that a line is unneccessary in the code.) 😄